components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/jwt/JWTValidatorImpl.java

```diff
@@ -91,7 +92,32 @@ public JWTValidationInfo validateToken(SignedJWTInfo signedJWTInfo) throws APIMa
             throw new APIManagementException("Error while parsing JWT", e);
         }
     }
+    public boolean isValidCertificateBoundAccessToken(SignedJWTInfo signedJWTInfo) { //Holder of Key token
```

Comment:
make protected since this method no need to expose

Reply:
fixed with 8ecefe3b77c909534ee7c401674e1c0d7459ca55